### DISCUSSION TITLE
RAUC install bundle stuck a 99%

### DISCUSSION THREAD

**Discussion by ghost** [rauc/rauc#1550] [General]
We are using the busctl  command to start an OTA of the rauc bundle.
Please see the command below:
```
busctl call de.pengutronix.rauc / de.pengutronix.rauc.Installer InspectBundle sa{sv} "http://192.168.0.21:8000/update-bundle-imx8mnevk.raucb" 0
```

From the above we have noticed 1 in 10 OTA's the busctl get-property Progress shows 99%; but the range server reports that the installation is successfull. Upon next power cycle of the hardware; the system successfully boot the correct slot when the OTA was performed.

Please see below the busctl commands used to determine the progress of the OTA
```
busctl get-property de.pengutronix.rauc / de.pengutronix.rauc.Installer Progress
(isi) 99 "Updating slots done." 2

busctl get-property de.pengutronix.rauc / de.pengutronix.rauc.Installer Operation
s "installing"

busctl get-property de.pengutronix.rauc / de.pengutronix.rauc.Installer LastError
s ""
```

**Comment by ejoerns** [comment]
@pritpalm-kuk, Could you send a log of the RAUC service from the target that reports this 99%?
